notch8/atla_digital_library
# fix oai ia imports

# Coauthor
@aprilrieger

# Summary
imports with the oai ia parser were failing. This PR upgrades some of the overridden bulkrax files to fix these imports. 

# Related
https://github.com/scientist-softserv/atla_digital_library/issues/248

# Screenshot
can now import successfully with the OaiIA parser: 
<img width="1103" alt="image" src="https://user-images.githubusercontent.com/73361970/233202067-149fee96-7557-4b91-89ef-b8879a49b090.png">


# Acceptance Criteria
- [ ] as a user, I can import an oai feed with the oai ia parser

# Testing Instructions
1. Sign in and go to the /importers page
2. Click the button to create a new importer
3. Use the following applicable info to fill out the importer:

name: `AMBS -  Biblioteca Digital Anabautista`
admin_set_id: `admin_set/default`
limit: `25` <---- set this or you will get ALOT of entries
parser: `OAI - Internet Archive`
base_url: `https://archive.org/services/oai.php`
metadata_prefix: `oai_dc`
set: `collection:bibliotecadigitalanabautista`
collection_title: `AMBS -  Biblioteca Digital Anabautista`
institution_name: `Anabaptist Mennonite Biblical Seminary`
thumbnail_url: `https://archive.org/services/img/<%= identifier.split(':').last %>"}`

4. Ensure the importer says "completed" and works import successfully, report any errors you see with screenshots and behavior description, try to include as much detail as possible so we can pickup where you left off if it gets kicked back :)
